format/Schema.fbs

```diff
@@ -44,9 +44,14 @@ table Null {
 table Struct_ {
 }
 
+/// Same as List, but with 64-bit offsets, allowing to represent
```

Comment:
this comment seems like it is in the wrong place?

Reply:
Uh, right, will fix :-)